helpers/AppBase.ts

```diff
@@ -125,3 +125,7 @@ export class App {
     return [...fullMatches, ...partialMatches];
   }
 }
+
+export const preprocessKeybind = (keybind: string) => {
```

Comment:
:zap: **Suggestion** 
Consider using a more descriptive name for the placeholder string '???' to improve code readability and maintainability.